react-app/src/features/submission/app-k/index.tsx

```diff
@@ -10,24 +8,22 @@ import { SubmissionServiceParameters, submit } from "@/api/submissionService";
 import { useGetUser } from "@/api/useGetUser";
 import { Authority } from "shared-types";
 import { useMutation } from "@tanstack/react-query";
-import { useNavigate } from "@/components/Routing";
 import { useEffect, useState } from "react";
-import * as Content from "@/components";
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import { SlotAdditionalInfo, SlotAttachments } from "@/features";
 import { documentPoller } from "@/utils/Poller/documentPoller";
 import { SubmitAndCancelBtnSection } from "../waiver/shared-components";
+import { getFormOrigin } from "@/utils";
 
 export const AppKSubmissionForm = () => {
-  const nav = useNavigate();
+  const navigate = useNavigate();
```

Comment:
I like this. The verb `navigate` is more descriptive and clear than the ambiguous `nav`